index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
+            <div class="canop">
+                <p class="can-info">CANOPY</p>
+                <div class="point"></div>
+                <p class="back">Back End Dev</p>
+                <div class="point"></div>
+                <p class="back">2015</p>
+            </div>
+            <div class="languages">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see">See project</button>
```

Comment:
- [ ] The button you have implemented does not have the proper styles as per the design specifications. Please carefully review the design and update your code to match the specified styles for the default, hover, and active states of the button.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/48276175-1c5f-40c0-ba49-4ffa8d4c9187)
- [ ] Additionally, the vertical spacing (top and bottom) of the button does not align with the design. I recommend adjusting the spacing to match the design as closely as possible.
Please make these adjustments to ensure that the button appearance and spacing align with the design requirements. Great work so far, and keep it up! 👍🌟